src/utils/spec-parser.js

```diff
@@ -155,6 +158,77 @@ function groupByPaths(openApiSpec) {
 }
 */
 
+function getComponents(openApiSpec) {
+  const components = [];
+  for (const component in openApiSpec.components) {
```

Comment:
must ensure `openApiSpec.components` is not `undefined`

Reply:
done